Formula/balena-cli.rb

```diff
@@ -4,8 +4,8 @@ class BalenaCli < Formula
   desc "The official balena CLI tool"
   homepage "https://www.balena.io/docs/reference/cli/"
   # balena-cli should only be updated every 10 releases on multiples of 10
```

Comment:
Do we not do this anymore?

Reply:
The throttle audit only looks at the last part of the version string, and 0 is a multiple of 10. The "every 10 releases" is a little misleading.

Reply:
unless balena-cli does not respect the semver entirely, the throttle might still make sense.

With that being said, they did the two minor version releases in one day though.